app/src/androidTest/java/com/github/h3lp3rs/h3lp/MainPageTestActivity.kt

```diff
@@ -213,12 +213,12 @@ class MainPageTestActivity : H3lpAppTest() {
                 )
             }
         ) {
-            val uiDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
-            // Should immediately receive a notification
-            uiDevice.wait(Until.hasObject(By.textStartsWith("H3LP")), 3000)
-            val notification =
-                uiDevice.findObject(By.text(globalContext.getString(R.string.emergency)))
-            assertNull(notification)
+//            val uiDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
```

Comment:
Forgotten comments ?

Reply:
Actually the test can no longer pass since we remove the distance filter, but we should keep it for later use (will comment that)